.github/workflows/coverage.yml

```diff
@@ -2,10 +2,14 @@ name: Coverage
 
 on:
   push:
-    branches: [main]
+    branches:
+      - main
+      - "*/*"
```

Comment:
I think we basically want to capture every branches here no? In which case we want `*/**` or `**` I believe

Reply:
I was thinking that we could only build branches following the conventional commit, but as you wish

Reply:
Yeah, sometimes we are naming branches without slash for very quick changes without passing by linear tickets, so I guess we just want to run the main jobs for any non-draft PR regardless of the branch name